contracts/ERC20/ERC20ProxyStorage.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.5.13;
+
+/**
+ * All storage must be declared here
+ * New storage must be appended to the end
+ * Never remove items from this list
+ */
+contract ERC20ProxyStorage {
+
+    uint256 constant MAX_SUPPLY = 145000000000000000;
+
+    mapping (address => uint256) _balances;
+    mapping (address => mapping (address => uint256)) _allowances;
+    uint256 _totalSupply;
+
+    address owner_;
+    address pendingOwner_;
+
+    /* Additionally, we have several keccak-based storage locations.
```

Comment:
Are we actually using keccak-based storage in this repository? If not, we can safely remove this comment.